tests/test_jointcal_cfht.py

```diff
@@ -68,8 +68,8 @@ def test_jointcalTask_2_visits(self):
         pa1 = 0.014
         metrics = {'collected_astrometry_refStars': 825,
                    'collected_photometry_refStars': 825,
-                   'selected_astrometry_refStars': 825,
-                   'selected_photometry_refStars': 825,
+                   'selected_astrometry_refStars': 350,
+                   'selected_photometry_refStars': 350,
```

Comment:
Is it expected that jointcal can only use a smallish fraction of the reference stars?

Reply:
I've never dug into it, but I'm not surprised: the refstar list comes from `loadSkyCircle`, which would give stars outside the bbox of the input exposures. That plus the cuts imposed by `astrometrySourceSelector` mean I'd be very surprised if even a majority of refStars are used in these tests (especially given the not-full-focal plane test data).